config/routes.rb

```diff
@@ -1,12 +1,10 @@
-# frozen_string_literal: true
-
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get 'up' => 'rails/health#show', as: :rails_health_check
-
   # Defines the root path route ("/")
-  # root "posts#index"
+  # root "articles#index"
+  resources :users, only: [:show, :index] do
+    resources :posts, only: [:show, :index,] do
+      end
+   end
```

Comment:
- [ ] [Optional] Great work so far on updating your code. However, there are still some changes that need to be made. In the resources :posts block, the :index action has an extra comma at the end of the only array on line number 7. While this may not cause an error in Ruby, it could be a typo or unnecessary. Additionally, since there are no routes inside the post, the do block is unnecessary. You can modify it using the following approach.
```suggestion
  resources :users, only: [:show, :index] do
    resources :posts, only: [:show, :index]
  end
```